templates/aws/nodes/manifest.yaml

```diff
@@ -56,6 +64,10 @@ variables:
     type: string
     optional: true
     description: "Public IP address of the bastion node"
+  airgap_setup:
```

Comment:
why is this showing up as a new addition? We already have this, is your git history up to date?
https://github.com/rancherlabs/corral-packages/blob/5f0e0d10bcc4b2e84760817d77ad1432fcf564e0/templates/aws/cluster_nodes/manifest.yaml#L56C7-L56C7

Reply:
Why is it showing as a new addition? Something is off.

Reply:
Nevermind Izaac I was looking at the wrong templated sorry! Can you fix the description, to be this instead?
"Boolean that when set, will create airgap nodes rather regular cluster nodes."

Reply:
Sure, updated.